docs/source/cli.rst

```diff
@@ -5,9 +5,9 @@
 Command Line Interface
 ======================
 
-When git-machete is installed it is added to your path, so it can be called from command line: ``git-machete <command> <options>``. git-machete comes with a wide variety of commands and their options and in this site you can find their documentation.
+When git machete is installed, it adds ``machete`` command to ``git``, so it can be called from command line: ``git machete <command> <options>``. git machete comes with a wide variety of commands and their options and in this site you can find their documentation.
```

Comment:
Sorry for not making it clear, `git-machete` is the name of the software itself but it's meant to be invoked `git machete <command>` from the shell

Reply:
(even though `git-machete <command>` is also valid)